<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="3159" started-at="2018-09-23T09:09:07Z" finished-at="2018-09-23T09:09:10Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="3159" started-at="2018-09-23T09:09:07Z" finished-at="2018-09-23T09:09:10Z">
      <class name="testpackage.NewTest">
        <test-method status="FAIL" signature="test1()[pri:0, instance:testpackage.NewTest@45820e51]" name="test1" duration-ms="3128" started-at="2018-09-23T09:09:07Z" finished-at="2018-09-23T09:09:10Z">
          <exception class="org.sikuli.script.FindFailed">
            <message>
              <![CDATA[C:/Users/Pasindu/Documents/Sikuli.sikuli/chrome.png: (50x45) in S(0)[0,0 1366x768] E:Y, T:3.0]]>
            </message>
            <full-stacktrace>
              <![CDATA[FindFailed: C:/Users/Pasindu/Documents/Sikuli.sikuli/chrome.png: (50x45) in S(0)[0,0 1366x768] E:Y, T:3.0
  Line 2279, in file Region.java

at org.sikuli.script.Region.wait(Region.java:2279)
at org.sikuli.script.Region.wait(Region.java:2297)
at org.sikuli.script.Region.getLocationFromTarget(Region.java:3253)
at org.sikuli.script.Region.doubleClick(Region.java:3907)
at org.sikuli.script.Region.doubleClick(Region.java:3892)
at testpackage.NewTest.test1(NewTest.java:15)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.sikuli.script.FindFailed -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
      </class> <!-- testpackage.NewTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
